src/webswview/lkswview/api/views.py

```diff
@@ -16,4 +25,64 @@ def get(self, id):
         return {'task': 'Say "Hello, World!"'}
 
 
+class SrcPackage(Resource):
+    def get(self, repo_name, suite_name, name):
+        with session_scope() as session:
+            rss = repo_suite_settings_for(session, repo_name, suite_name, fail_if_missing=False)
+            if not rss:
+                abort(404)
+
+            spkgs = (
+                session.query(SourcePackage)
+                .options(undefer(SourcePackage.version))
+                .filter(SourcePackage.repo_id == rss.repo_id)
+                .filter(SourcePackage.suites.any(ArchiveSuite.id == rss.suite_id))
+                .filter(SourcePackage.name == name)
+                .filter(SourcePackage.time_deleted.is_(None))
+                .order_by(SourcePackage.version.desc())
+                .all()
+            )
+            if not spkgs:
+                abort(404)
+
+            repo = rss.repo
+            suite = rss.suite
+
+            return {
+                'pkgs': [
+                    {
+                        'uuid': spkg.uuid,
+                        'source_uuid': spkg.source_uuid,
+                        'name': spkg.name,
+                        'version': spkg.version,
+                        # skipping repo because presumably it's same as rss.repo
+                        # skipping suites because there's no good reason to have them in a suite-specific query
```

Comment:
...unless the API user wants to know what *other* suites this package in its particular version is also present in for the given repository. Having that info may (unlike repo) actually be interesting here.